indigo_api/models.py

```diff
@@ -1168,6 +1170,8 @@ class Meta:
 
     labels = models.ManyToManyField('TaskLabel', related_name='+')
 
+    extra_data = JSONField(default={'points': 12})
```

Comment:
This indigo package shouldn't need to know about the points at all. This can just be an empty JSONField. (Same applies to the migration).